### ISSUE TITLE
Use curly braces check in ShellCheck 0.7.0

### ISSUE THREAD

**Issue by Frenzie** [koreader/koreader#5152]
* KOReader version: 2019.07

When upgrading, look into the new optional checks. See https://www.vidarholen.net/contents/blog/?p=793

@NiLuJe has a strong preference for this one. ;-)

```
$ cat foo
#!/bin/sh
# shellcheck enable=require-variable-braces
name=World
echo "Hello $name"

$ shellcheck foo
In foo line 4:
echo "Hello $name"
            ^---^ SC2250: Prefer putting braces around variable references even when not strictly required.

Did you mean:
echo "Hello ${name}"
```

**Comment by NiLuJe** [comment]
I do like me some curly braces indeed ;p.

**Comment by Frenzie** [comment]
This is still open? Fixed in #5809.
